scripts/Linux-Fetch.sh

```
@@ -48,5 +48,11 @@ VENDOR=$(realpath $LUX_DIR/Core/vendor/)
 	chmod +x $LUX_DIR/premake5
 
 ## Vulkan SDK
-	curl -L https://sdk.lunarg.com/sdk/download/$VULKAN_VERSION/linux/vulkansdk-linux-x86_64-$VULKAN_VERSION.tar.xz | tar -x -J -f - -C $VENDOR
-	mv $VENDOR/$VULKAN_VERSION $VENDOR/VulkanSDK
+	# Extract into an isolated tmp dir, not straight into $VENDOR: if $VENDOR/VulkanSDK already
+	# exists (even as unrelated leftover cruft), `mv` treats it as a move-into rather than a
+	# rename, silently nesting the SDK one level too deep.
+	VK_TMP=$(mktemp -d)
+	curl -L https://sdk.lunarg.com/sdk/download/$VULKAN_VERSION/linux/vulkansdk-linux-x86_64-$VULKAN_VERSION.tar.xz | tar -x -J -f - -C $VK_TMP
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)